MapboxVisionTests/RecordCoordinatorTests.swift

```diff
@@ -72,6 +70,83 @@ final class RecordCoordinatorTests: XCTestCase {
         XCTAssert(directoryExists(at: recordingPath), "Recording should be saved at \(recordingPath) after recording is stopped")
     }
 
+    private class TestClipRequest {
+        let start: Float
+        let end: Float
+
+        init(start: Float, end: Float) {
+            self.start = start
+            self.end = end
+        }
+    }
+
+    private class RecordDelegate: RecordCoordinatorDelegate {
+        private let onRecordingStopped: () -> Void
+
+        init(onRecordingStopped: @escaping () -> Void) {
+            self.onRecordingStopped = onRecordingStopped
+        }
+
+        func recordingStarted(path: String) {
+        }
+
+        func recordingStopped(recordingPath: RecordingPath) {
+            onRecordingStopped()
+        }
+    }
+
+    private func performVideoClipping(with requests: [TestClipRequest], _ message: String = "", line: UInt = #line) {
```

Comment:
Could you please answer the previous comment:
> Also what's the point in line argument? Don't we want to know where exactly assert failed instead of where this function was called?

Reply:
> It's a helper method, the test in which we interested uses it
We want to know that the test failed, not its underlying implementation

Reply:
I think this implementation is a part of the test and it's interesting which particular assert has failed. But it's up to you.